### PR TITLE
Use +RTS -N2 -A16m to improve bulk sync performance

### PR DESCRIPTION
**Pull request by karknu**

### CODE REVIEW

File: cardano-node/cardano-node.cabal [hunk 1]
```diff
@@ -140,7 +140,7 @@ executable cardano-node
   ghc-options:         -threaded
                        -Wall
                        -rtsopts
-                       "-with-rtsopts=-T -I0"
+                       "-with-rtsopts=-T -I0 -N2 -A16m"
```
Comment (CodiePP) [inline cardano-node/cardano-node.cabal]: I am not sure this will help. The node is run in various scripts with different RTS flags.
Do we need this here?

Comment (karknu) [reply cardano-node/cardano-node.cabal]: Yes, it needs to be run with those flags.
But if it is bypassed in scripts somewhere it doesn't do much good.

**Comment by oneEdoubleD** [comment]
This was _fast_ - 1h6m for a complete sync of Staging (170 epochs). Previously it was ~6h30m, and the best I've ever actually seen was ~2h30m.

Built with:
`nix-build -A scripts.staging.node`

File: cardano-node/cardano-node.cabal (hunk 1 above)
Comment (mrBliss) [reply cardano-node/cardano-node.cabal]: In my experiments, having `-N2` (or greater) has been very important for performance.

**Pull request merged**